README.md

```diff
@@ -102,18 +102,30 @@ When an unsuccessful message displays, you can troubleshoot the installation by
 After you install the plug-in, you create a Db2 profile. A Db2 profile is required to issue commands to the Db2 region. Db2 profiles contain your host, port, user name, password, and a database name for the IBM Db2 server of your choice. You can create multiple profiles and switch between them as needed.
 
 **Follow these steps:**
-1.  Create a Db2 profile: 
+1.  Install the IBM Db2 Plug-in for Zowe CLI
+2.  Create a Db2 profile: 
     
     ```
-    zowe profiles create db2 <profile name> -H <host> -P <port> -u <user> -p <password> -d <database>
+    zowe config init
+    ```
+3.  Set the port number to your Db2 port:
+
+    ```
+    zowe config set profiles.db2.properties.port <port number>
+    ```
+
+4.  Set the database to your Db2 database:
```

Comment:
data base should be two words.

Reply:
I do not believe database should be two words. It is referenced as `database` in Zowe Docs.